### PR TITLE
Contact Me Section Review

### PR DESCRIPTION
**Pull request by Nedramrez**
Add Contact Me Section HTML code
Partner's commit: Style Contact Me Section in a style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,66 +199,97 @@ <h6>Back End Dev</h6>
...
+              you need built or a project that needs coding. I’d love to help with it
+            </p>
+          </div>
+          <div class="container2">
+            <form action="https://formspree.io/f/xgedddaq" method="POST">
+              <input type="text" name="name" maxlength="30" placeholder="Your name" required />
+              <input type="email" name="email" placeholder="Email address" required />
+              <textarea name="message" maxlength="500" placeholder="Enter your message here" required>
+              </textarea>
+              <input class="button" type="submit" value="Get In Touch">
```
Comment (RolandM99) [inline index.html:288]: - [x] Kindly, make sure that Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. check the screenshot below:
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Nedramrez, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Nedramrez) [reply index.html:288]: Thank you very much for your friendly and professional review!

**Review by Ismailco — APPROVED** [review]
### Hi @Nedramrez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nedramrez** [comment]
> ### Hi @Nedramrez,
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
> 
> cr-approve [ cr-approve ](https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif) [ ](https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif)
> 
> ## Highlights
> heavy_check_mark No linter errors heavy_check_mark Used Github Flow heavy_check_mark Work documented professionally
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you!

**Comment by Nedramrez** [comment]
@Ismailco Thank you!

**Pull request merged**